forte/processors/base/data_augment_processor.py

```diff
@@ -46,44 +55,559 @@ class ReplacementDataAugmentProcessor(BaseDataAugmentProcessor):
     levels: character, word, sentence or document.
     """
     def __init__(self):
-        """
-        The replaced_spans records the entries replaced by new texts.
-        It consists of tuples (entry, new text) inserted by :func: replace.
-        The new text will be used for building new data pack.
+        r"""
+        - replaced_annos: {datapack id: List[span, text]}
```

Comment:
These instance attributes are not part of the `__init__` parameter, you should directly add these comments inline of the variable, instead of writing at the function level. The user don't need to know these details to use this class.